rasa_nlu/components.py

```diff
@@ -49,7 +49,7 @@ class MissingArgumentError(Exception):
 
     def __init__(self, message):
         # type: (str) -> None
```

Comment:
Accordning to https://www.python.org/dev/peps/pep-0484/#the-meaning-of-annotations the `__init__` functions ought to be annotated with `None` as a return type.